### PR TITLE
add routes and html structure

### PR DESCRIPTION
**Pull request by shahzaman900**
### add routes and html structure

- Add routes for users 
- create HTML structure for users 
- create post against user

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+      <div class="user">
+        <img class="user-img" src="<%= user.photo %>" alt="User photo">
+        <div class="name-posts-count">
+            <h2 class="user-name"><%= link_to user.name, user_path(user) %></h2>
+
+            <div class="user-posts-count"><p>Number of posts: <%= user.post_counter %></p></div>
+        </div>
+      </div>
+    <% end %>
+</div>
```
Comment (Qoosim) [inline app/views/users/index.html.erb:12]: - [ ] You're required to have the same form for posts in the view. Kindly do just that.

### Project Requirement.
view

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <% else %>
+        <div id="user-recent-posts">
+            <% @user.most_recent_posts.each_with_index do |post, i| %>
+                <%= render partial: "posts/post_data", locals: { post: post, i: i + 1 } %>
+            <% end %>
+        </div>
+
+        <div id="all-posts"><a href="<%= user_posts_path(@user) %>">See all posts</a></div>
+    <% end %>
+</div>
```
Comment (Qoosim) [inline app/views/users/show.html.erb:28]: - [ ] You're required to have the same form for posts in the view. Kindly do just that.

### Project Requirement.
view

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @sheezu900, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Implemented some project requirements. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @sheezu900 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**